thirdparty/README.md

```diff
@@ -791,7 +791,7 @@ File extracted from upstream release tarball:
 ## xatlas
 
 - Upstream: https://github.com/jpcy/xatlas
-- Version: git (16ace528acd2cf1f16a7c0dde99c42c486488dbe, 2022)
+- Version: git (f700c7790aaa030e794b52ba7791a05c085faf0c, 2022)
```

Comment:
There's no change upstream between those two commits for the files we include. So I just bump it in the docs so I stop having to check it out to find again that we're already up to date.